docs/tutorials/dqn.rst

```diff
@@ -51,12 +53,13 @@ For the demonstration, here we use the second block of codes.
 
     Otherwise, the outputs of these envs may be the same with each other.
 
+
 .. _build_the_network:
 
 Build the Network
 -----------------
 
-Tianshou supports any user-defined PyTorch networks and optimizers but with the limitation of input and output API. Here is an example code: 
+Tianshou supports any user-defined PyTorch networks and optimizers only with the limitation of input and output API. Here is an example:
```

Comment:
"Tianshou supports any user-defined PyTorch networks and optimizers. Yet, of course, the inputs and outputs must comply with Tianshou's API. Here is an example:"